src/types/Format.ts

```diff
@@ -5,13 +5,15 @@ const Format = t.Union(
   t.Literal('png'),
   t.Literal('gif'),
   t.Literal('webp'),
+  t.Literal('svg'),
 );
 
 const Mime = t.Union(
   t.Literal('image/jpeg'),
   t.Literal('image/png'),
   t.Literal('image/gif'),
   t.Literal('image/webp'),
+  t.Literal('image/svg+xml')
```

Comment:
Since mime and format does not match each other, we need to convert them manually in `formatToMime`:
```
if (typedFormat === 'svg') {
  return ('image/svg+xml') as Mime;
}
```
And a similar change is needed in `mimeToFormat`.